test/resources/3scale_api.go

```diff
@@ -203,3 +208,119 @@ func (r *ThreeScaleAPIClientImpl) CreateProduct(name string) (string, error) {
 	id := strings.Split(href, "/")
 	return id[len(id)-1], nil
 }
+
+func (r *ThreeScaleAPIClientImpl) SetUserAsAdmin(username string, email string, userID string) error {
+
+	url := fmt.Sprintf("%v/p/admin/account/users/"+userID, r.host)
+	formUrl := fmt.Sprintf("%v/p/admin/account/users/%v/edit", r.host, userID)
```

Comment:
I think this could be done by hitting the 3scale API, you can see an example in the 3scale reconciler or just reuse the existing 3scale client https://github.com/integr8ly/integreatly-operator/blob/master/pkg/products/threescale/three_scale_client.go#L167

Reply:
As discussed, the approach taken here was to follow an existing precedent i.e. the approach taken in 3scale_crudl.go